docs/default-theme-config/README.md

```diff
@@ -48,6 +48,25 @@ module.exports = {
 }
 ```
 
+The item in `themeConfig.nav` could also be a dropdown menu:
```

Comment:
_Those links can also be dropdown menus, add nested items via `items`:_

Reply:
fixed at 50689ea13da744b0fdac8368e815fd2e6bdd7a30, thanks